backend/models.py

```diff
@@ -0,0 +1,57 @@
+
+# Create your models here.
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+import uuid
+
+
+class User(AbstractUser):
+    # Using: username, password, first_name, last_name, email inherited from Abstractuser
+
+
+class Post(models.Model):
+    postId = models.UUIDField(
+        primary_key=True, default=uuid.uuid4, editable=False, unique=True)
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    title = models.CharField(max_length=50)
+    timestamp = models.DateTimeField(auto_now_add=True)
+    description = models.CharField(max_length=450)
```

Comment:
Instead of calling it `description`, call  it  sth like `content` to be a little bit more descriptive. Also, change `CharField` to `TextField` as max length is not required for `Textfield`

Reply:
Completed!